app/services/api/v2/suggestions_service.rb

```diff
@@ -18,10 +18,6 @@ def handle_commodity_record(commodity)
       def handle_search_reference_record(search_reference)
         Api::V2::SuggestionPresenter.new(search_reference.id, search_reference.title)
       end
-
-      def handle_chemical_record(chemical)
```

Comment:
We've stopped populating the typeahead with chemicals - this was just leftover cruft